extensions/amp-base-carousel/1.0/base-carousel.css.js

```diff
@@ -52,12 +52,57 @@ export const arrowPlacement = {
   // Center the button vertically.
   top: '50%',
   transform: 'translateY(-50%)',
+  alignItems: 'center',
 };
 
 export const defaultArrowButton = {
-  width: '32px',
-  height: '32px',
-  background: 'rgba(0, 0, 0, 0.25)',
+  position: 'relative',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  width: '36px',
+  height: '36px',
+  padding: 0,
+  margin: '12px',
+  backgroundColor: 'transparent',
+  border: 'none',
+  outline: 'none',
+  transition: '200ms stroke',
+};
+
+export const arrowBaseStyle = {
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  width: '100%',
+  height: '100%',
+  borderRadius: '50%',
+};
+
+export const arrowFrosting = {
+  ...arrowBaseStyle,
+  backdropFilter: 'blur(3px)',
+  backgroundColor: 'cyan',
+};
+
+export const arrowBackdrop = {
+  ...arrowBaseStyle,
+  backdropFilter: 'blur(12px) invert(1) grayscale(0.6) brightness(0.8)',
+  opacity: 0.5,
+};
+
+export const arrowBackground = {
+  ...arrowBaseStyle,
+  boxShadow: `0 0 0px 1px rgba(0, 0, 0, 0.08) inset,
+      0 1px 4px 1px rgba(0, 0, 0, 0.2)`,
+  transition: '200ms background-color',
+};
+
+export const arrowIcon = {
+  position: 'relative',
+  zIndex: 1,
```

Comment:
Is `zIndex` necessary here? A bit unexpected, tbh. At least if it was on the button itself it'd be more expected.